### PR TITLE
Prepare for release stable version for Azure.ResourceManager and Azure.ResourceManager.Resources.

### PR DESCRIPTION
**Pull request by ArthurMa1978**
Bingo~

### CODE REVIEW

File: doc/dev/mgmt_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
+
+# Quickstart Tutorial - Resource Management
```
Comment (ArthurMa1978) [inline doc/dev/mgmt_quickstart.md:2]: This quickstart will add more samples after release other ring 0 RPs

File: sdk/resourcemanager/Azure.ResourceManager/samples/Sample3_CreatingAVirtualNetwork.md [hunk 2]
```diff
@@ -6,12 +6,13 @@ In this example, we'll create a virtual network. Since the SDK follows the resou
 ## Import the namespaces
 These are the namespaces needed for this project:
 ```C#
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
```
Comment (ArthurMa1978) [inline sdk/resourcemanager/Azure.ResourceManager/samples/Sample3_CreatingAVirtualNetwork.md:11]: The network samples will be updated once Network sdk released

File: doc/dev/mgmt_quickstart.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+released now. Those packages share a number of new features
+such as Azure Identity support, HTTP pipeline, error-handling.,etc, and
+they also follow the new Azure SDK guidelines which create easy-to-use
+APIs that are idiomatic, compatible, and dependable.
+
+You can find the details of those new libraries
+[here](https://azure.github.io/azure-sdk/releases/latest/mgmt/dotnet.html)
+
+In this basic quickstart guide, we will walk you through how to
+authenticate to Azure using the preview libraries and start interacting
```
Comment (Yao725) [inline doc/dev/mgmt_quickstart.md:14]: should remove the word `preview`

File: doc/dev/mgmt_quickstart.md [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+More information and different authentication approaches using Azure Identity can be found in [this document](https://docs.microsoft.com/dotnet/api/overview/azure/identity-readme?view=azure-dotnet).
+
+## Authenticate the Client
+
+The default option to create an authenticated client is to use `DefaultAzureCredential`. Since all management APIs go through the same endpoint, in order to interact with resources, only one top-level `ArmClient` has to be created.
+
+To authenticate to Azure and create an `ArmClient`, do the following:
+
+```C# Snippet:Readme_AuthClient
+using System;
```
Comment (Yao725) [inline doc/dev/mgmt_quickstart.md:43]: I think we'd better sort the using

File: doc/dev/mgmt_quickstart.md [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+Additional documentation for the `Azure.Identity.DefaultAzureCredential` class can be found in [this document](https://docs.microsoft.com/dotnet/api/azure.identity.defaultazurecredential).
+
+## Key concepts
+### Understanding Azure Resource Hierarchy
+
+To reduce both the number of clients needed to perform common tasks and the amount of redundant parameters that each of those clients take, we have introduced an object hierarchy in the SDK that mimics the object hierarchy in Azure. Each resource client in the SDK has methods to access the resource clients of its children that is already scoped to the proper subscription and resource group.
+
+To accomplish this, we're introducing 3 standard types for all resources in Azure:
+
+### **[Resource].cs**
```
Comment (Yao725) [inline doc/dev/mgmt_quickstart.md:65]: It should be `[Resource]Resource.cs`, though it looks a bit strange.

File: doc/dev/mgmt_quickstart.md [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+Console.WriteLine($"Subscription: {id.SubscriptionId}");
+Console.WriteLine($"ResourceGroupResource: {id.ResourceGroupName}");
+Console.WriteLine($"Vnet: {id.Parent.Name}");
+Console.WriteLine($"Subnet: {id.Name}");
+```
+
+## Managing Existing Resources By Id
+Performing operations on resources that already exist is a common use case when using the management client libraries. In this scenario you usually have the identifier of the resource you want to work on as a string. Although the new object hierarchy is great for provisioning and working within the scope of a given parent, it is not the most efficient when it comes to this specific scenario.  
+
+Here is an example how you to access an `AvailabilitySet` object and manage it directly with its id:
```
Comment (Yao725) [inline doc/dev/mgmt_quickstart.md:148]:
```suggestion
Here is an example how you can access an `AvailabilitySet` object and manage it directly with its id: 
```

File: doc/dev/mgmt_quickstart.md [hunk 7]
```diff
@@ -0,0 +1,340 @@
...
+## Need help?
+
+- File an issue via [Github
+  Issues](https://github.com/Azure/azure-sdk-for-net/issues) and
+  make sure you add the "Preview" label to the issue
+- Check [previous
+  questions](https://stackoverflow.com/questions/tagged/azure+.net)
+  or ask new ones on StackOverflow using azure and .NET tags.
+
+## Contributing
```
Comment (Yao725) [inline doc/dev/mgmt_quickstart.md:322]: Unfortunately we forgot to update the contributing section here in this [PR](https://github.com/Azure/azure-sdk-for-net/pull/27947), can we do it now?

File: doc/dev/mgmt_quickstart.md (hunk 4 above)
Comment (ArthurMa1978) [reply doc/dev/mgmt_quickstart.md:43]: more like here `System` are base of every Azure libs

Comment (ArthurMa1978) [reply doc/dev/mgmt_quickstart.md]: just remove `System` as it's useless to this sample

File: doc/dev/mgmt_quickstart.md (hunk 7 above)
Comment (ArthurMa1978) [reply doc/dev/mgmt_quickstart.md:320]: fixed

**Review by m-nash — APPROVED** [review]

**Pull request merged**